File: app/src/main/java/me/blog/korn123/easyphotomap/helper/EasyPhotoMapApplication.java
package me.blog.korn123.easyphotomap.helper;

import android.support.multidex.MultiDexApplication;

import io.realm.Realm;

/**
 * Created by CHO HANJOONG on 2017-09-12.
 */

public class EasyPhotoMapApplication extends MultiDexApplication{
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
    }
}

File: app/src/main/java/me/blog/korn123/easyphotomap/helper/EasyPhotoMapApplication.kt
package me.blog.korn123.easyphotomap.helper

import android.support.multidex.MultiDexApplication

import io.realm.Realm

/**
 * Created by CHO HANJOONG on 2017-09-12.
 */

class EasyPhotoMapApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
    }
}
